proceeding/text/idea.tex

```diff
@@ -18,3 +18,21 @@ \section{Idea}
 \item Open source, open data - allows for 1. users to download any data from our website, and 2. for other developers to contribute to the code.
 
 \end{itemize}
+
+
+    gamma-sky.net is a one-stop resource for perusing images and catalogs but also closely examining a specific gamma-ray source.
+    Although it was mainly built for the astronomical community, the webpage also targets the general public through a
+    user-friendly interface and a clean information layout, all of which is compiled under cutting-edge web tools.
+
+    Individuals who access the website via any modern internet browser will be welcomed with the Map View page.
+    We overlaid multi-wavelength survey images, most of which are all-sky images, onto a three-dimensional sphere.
```

Comment:
I'm wondering if we should mention Aladin Lite here already?
"We overlaid ..." sounds like we implemented this, but in reality we just copy & pasted a few lines of Javascript and got all this functionality from Aladin Lite.
So either mention Aladin Lite here, or if you don't want to mix features and implementation, then drop the "we" here and instead rephrase e.g. like "The map view shows ..." to make it less sound like we implemented this.